interface/patient_file/summary/weno_fragment.php

```diff
@@ -0,0 +1,85 @@
+<?php 
+
+/**
+ * Weno Fragment.
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Kofi Appiah <kkappiah@medsov.com>
+ * @copyright Copyright (c) 2023 Kofi Appiah <kkappiah@medsov.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require "../../globals.php";
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Twig\TwigContainer;
+
+$twig = new TwigContainer(null, $GLOBALS['kernel']);
+
+if(!AclMain::aclCheckCore('patients', 'med')){
+        exit;
+}
+
+$sql = "SELECT * FROM prescriptions WHERE patient_id = ? " .
+        "AND indication IS NOT NULL";
+$res = sqlStatement($sql,array($pid));
+
+function getProviderByWenoId($external_id){
+        $provider = sqlQuery("SELECT fname, mname, lname FROM users WHERE weno_provider_id =", array($external_id));
+        if($provider){
+                return $provider['fname'] . " " . $provider['mname'] . " " . $provider['lname'];
+        } else{
+                return "";
+        }
+}
+?>
+<style>
+        .btr {
+                background-color: var(--primary);
+                padding: 1px;
+                color: white;
+                text-decoration: none;
+                border-radius: 10px;
+        }
+        .btr:hover {
+                color:white;
+                text-decoration: none;
+        }
+
+</style>
+
+
+
+<div class="row float-right mr-2">
+        <div class="mr-3 click" onclick="sync_weno()">
```

Comment:
where is the sync_weno() function?

Reply:
The sync_weno() function is in demographics.php. For some reason, placing it within the fragment does not work.